dev/AppNotifications/AppNotificationManager.cpp

```diff
@@ -93,8 +93,16 @@ namespace winrt::Microsoft::Windows::AppNotifications::implementation
         return m_activatedEventArgs;
     }
 
+    bool AppNotificationManager::IsSupported()
+    {
+        static bool isSupported{ !Security::IntegrityLevel::IsElevated() };
+        return isSupported;
+    }
+
     void AppNotificationManager::Register()
     {
+        THROW_HR_IF(HRESULT_FROM_WIN32(ERROR_NOT_SUPPORTED), !IsSupported());
```

Comment:
How does this throw surface at the API? We should use a RoOriginateError here so that we can include a string message explaining what about the call that isn't supported.

Reply:
Added some debug statements describing the failure reason.